src/integ-test/bdd-test/features/verifyPayamentReqTest.feature

```diff
@@ -2,10 +2,10 @@ Feature:  syntax checks for verifyPaymentReq
 
   Background:
     Given systems up
-      | name               | url                                                  | healtcheck      | service            |
-      | nodo-dei-pagamenti | http://localhost:8081                                | /monitor/health | /webservices/input |
-      | mock-ec            | http://localhost:8087/Pof                            | /api/v1/info    |                    |
-      | api-config         | http://localhost:8080/apiconfig/api/v1               | /info           |                    |
+      | name               | url                                                  | healthcheck      | service            |
```

Comment:
the table is useless now. Can we remove it?

Reply:
no, it is useful to generate headings in context table